src/swap/useSwapQuote.ts

```diff
@@ -182,6 +182,11 @@ function useSwapQuote(networkId: NetworkId, slippagePercentage: string) {
       }
 
       const quote: FetchQuoteResponse = await response.json()
+
+      if (!quote.unvalidatedSwapTransaction) {
+        throw new Error('No quote available')
```

Comment:
nit: can we hard code this as a const and share it in the swap screen so that we don't accidentally update this and break the functionality?

Reply:
Yes, I also thought about creating a new type of Error for this, but it felt like an overkill.
I'll create the constant 👌